### PR TITLE
Bump application-connector images

### PR DESCRIPTION
**Pull request by antoineco**
**Related issue(s)**

#6422

### CODE REVIEW

File: resources/application-connector/values.yaml [hunk 1]
```diff
@@ -87,5 +87,7 @@ tests:
     skipSslVerify: true
     image:
       dir:
+      # Prow job 'pre-kyma-tests-application-connector-tests" doesn't get
```
Comment (nachtmaar) [inline resources/application-connector/values.yaml]: Does a `/test pre-kyma-tests-application-connector-tests` help in order to create the image?

Comment (nachtmaar) [reply resources/application-connector/values.yaml]: To trigger the prow job manually from within a PR.

Comment (antoineco) [reply resources/application-connector/values.yaml]: I did that, but it will only create another (identical) PR image.

Comment (nachtmaar) [reply resources/application-connector/values.yaml]: True, normally only the post submit jobs create a real sha.

Comment (antoineco) [reply resources/application-connector/values.yaml]: It doesn't for this image, hence my comment.

Comment (nachtmaar) [reply resources/application-connector/values.yaml]: `post-kyma-tests-application-connector-tests` should do we trick, but you are right, it didn't run. At least I don't see it on `https://status.build.kyma-project.io/?type=postsubmit&job=post-kyma-tests-application-connector-tests`

**Pull request merged**